src/app/lib/views/play_control.js

```diff
@@ -189,8 +189,10 @@
                 lang: this.audio_selected
             };
 
-            var torrent = providers.torrent
-                .resolveStream(defaultTorrent, filters, this.model.attributes);
+            var torrent = providers.torrent.resolveStream
```

Comment:
Hello! I am trying to get my solution for this. I have removed this code entirely of the resolve stream, as I don't see where resolveStream is ever defined or used. Could it be inherited from a butter-desktop merge? As soon as I am authorised to create my PR I can link it here.

Reply:
This commit doesnt break the noprmal playing?

Reply:
tested it, if you mean just the browse and play then the answer is no.